openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/dynamic-form-field/dynamic-form-field.component.ts

```diff
@@ -237,6 +241,25 @@ export class DynamicFormFieldComponent implements OnInit, OnDestroy, AfterViewIn
     ).catch( error => console.info(`Scanning failed: ${error}`));
   }
 
+  isRequired(): boolean { 
+    return this.formField.required && !true;
```

Comment:
Will this code always return false?

Reply:
agreed, I'm guessing this should be something like 
```suggestion
    return this.formField.required && !this.formField. overridable;
```

Reply:
Ha, whoops. Yes, it will. Accidentally left some recent temp code in. This should have been `!this.formField.overridden`. Thanks for the catch, Robert, adjusted.

Reply:
Ha, whoops. Yes, it will. Accidentally left some recent temp code in. This should have been `!this.formField.overridden`. Thanks for the catch guys, adjusted.